[2025-04-18T11:25:58.836+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T11:25:58.908+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_etl_pipeline_mart.dbt_marts_core manual__2025-04-18T11:25:57.758337+00:00 [queued]>
[2025-04-18T11:25:58.923+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_etl_pipeline_mart.dbt_marts_core manual__2025-04-18T11:25:57.758337+00:00 [queued]>
[2025-04-18T11:25:58.925+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-18T11:25:58.941+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_marts_core> on 2025-04-18 11:25:57.758337+00:00
[2025-04-18T11:25:58.948+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dbt_etl_pipeline_mart', 'dbt_marts_core', 'manual__2025-04-18T11:25:57.758337+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/dbt_marts.py', '--cfg-path', '/tmp/tmp3cq48a8d']
[2025-04-18T11:25:58.950+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=663) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-18T11:25:58.950+0000] {standard_task_runner.py:105} INFO - Job 106: Subtask dbt_marts_core
[2025-04-18T11:25:58.951+0000] {standard_task_runner.py:72} INFO - Started process 664 to run task
[2025-04-18T11:25:59.146+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt_etl_pipeline_mart.dbt_marts_core manual__2025-04-18T11:25:57.758337+00:00 [running]> on host 9c8f13362ff5
[2025-04-18T11:25:59.391+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='azizjon' AIRFLOW_CTX_DAG_ID='dbt_etl_pipeline_mart' AIRFLOW_CTX_TASK_ID='dbt_marts_core' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T11:25:57.758337+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T11:25:57.758337+00:00'
[2025-04-18T11:25:59.392+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T11:25:59.410+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-18T11:25:59.412+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        export PATH=/home/airflow/.local/bin:$PATH\n        cd /opt/airflow/commerce_dbt\n        dbt run --select marts.core --profiles-dir .\n        ']
[2025-04-18T11:25:59.422+0000] {subprocess.py:99} INFO - Output:
[2025-04-18T11:26:01.798+0000] {subprocess.py:106} INFO - [0m11:26:01  Running with dbt=1.7.0
[2025-04-18T11:26:02.036+0000] {subprocess.py:106} INFO - [0m11:26:02  Registered adapter: postgres=1.7.0
[2025-04-18T11:26:02.443+0000] {subprocess.py:106} INFO - [0m11:26:02  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-04-18T11:26:02.448+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-04-18T11:26:02.449+0000] {subprocess.py:106} INFO - - seeds.initial_dataset
[2025-04-18T11:26:02.480+0000] {subprocess.py:106} INFO - [0m11:26:02  Found 12 models, 1 snapshot, 6 seeds, 43 tests, 6 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2025-04-18T11:26:02.485+0000] {subprocess.py:106} INFO - [0m11:26:02
[2025-04-18T11:26:02.804+0000] {subprocess.py:106} INFO - [0m11:26:02  Concurrency: 4 threads (target='dev')
[2025-04-18T11:26:02.804+0000] {subprocess.py:106} INFO - [0m11:26:02
[2025-04-18T11:26:02.813+0000] {subprocess.py:106} INFO - [0m11:26:02  1 of 4 START sql table model commerce_core.dim_customers ....................... [RUN]
[2025-04-18T11:26:02.814+0000] {subprocess.py:106} INFO - [0m11:26:02  2 of 4 START sql table model commerce_core.dim_products ........................ [RUN]
[2025-04-18T11:26:02.815+0000] {subprocess.py:106} INFO - [0m11:26:02  3 of 4 START sql table model commerce_core.dim_stores .......................... [RUN]
[2025-04-18T11:26:03.081+0000] {subprocess.py:106} INFO - [0m11:26:03  3 of 4 OK created sql table model commerce_core.dim_stores ..................... [[32mSELECT 5[0m in 0.26s]
[2025-04-18T11:26:03.088+0000] {subprocess.py:106} INFO - [0m11:26:03  1 of 4 OK created sql table model commerce_core.dim_customers .................. [[32mSELECT 1000[0m in 0.27s]
[2025-04-18T11:26:03.091+0000] {subprocess.py:106} INFO - [0m11:26:03  2 of 4 OK created sql table model commerce_core.dim_products ................... [[32mSELECT 17[0m in 0.27s]
[2025-04-18T11:26:03.094+0000] {subprocess.py:106} INFO - [0m11:26:03  4 of 4 START sql table model commerce_core.fct_orders .......................... [RUN]
[2025-04-18T11:26:03.215+0000] {subprocess.py:106} INFO - [0m11:26:03  4 of 4 OK created sql table model commerce_core.fct_orders ..................... [[32mSELECT 10000[0m in 0.12s]
[2025-04-18T11:26:03.269+0000] {subprocess.py:106} INFO - [0m11:26:03
[2025-04-18T11:26:03.270+0000] {subprocess.py:106} INFO - [0m11:26:03  Finished running 4 table models in 0 hours 0 minutes and 0.78 seconds (0.78s).
[2025-04-18T11:26:03.297+0000] {subprocess.py:106} INFO - [0m11:26:03
[2025-04-18T11:26:03.297+0000] {subprocess.py:106} INFO - [0m11:26:03  [32mCompleted successfully[0m
[2025-04-18T11:26:03.298+0000] {subprocess.py:106} INFO - [0m11:26:03
[2025-04-18T11:26:03.299+0000] {subprocess.py:106} INFO - [0m11:26:03  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-04-18T11:26:04.169+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-18T11:26:04.330+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T11:26:04.331+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dbt_etl_pipeline_mart, task_id=dbt_marts_core, run_id=manual__2025-04-18T11:25:57.758337+00:00, execution_date=20250418T112557, start_date=20250418T112558, end_date=20250418T112604
[2025-04-18T11:26:04.508+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T11:26:04.591+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T11:26:04.595+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
